admin/installer/postgres/docker-compose-node1.yml

```diff
@@ -0,0 +1,12 @@
+postgres:
+  image: postgres
+  ports:
+     -  "5432:5432"
+  environment:
+     - "constraint:flocker-node==2"
+     - "POSTGRES_USER=flocker"
+     - "POSTGRES_PASSWORD=flocker"
```

Comment:
:grey_question: Setting these here is a bit concerning to me (although not quite as concerning as the MySQL one). It seems conceivable to me that people scan the internet for postgress and mysql databases running on the standard ports, and they might poke around for insecure passwords and usernames. Consider doing something to force the user to provide/enter these. (If you don't set them to anything, what happens?)